scripts/tools/tools-object.js

```diff
@@ -78,10 +79,10 @@ async function convertTools(data) {
             }
           });
         } else {
+          console.error('Script is not failing, it is just dropping errors for further investigation');
```

Comment:
moved it up as when I was testing it locally, it was strange to see this at the end of logs, not intuitive.